src/clusterfuzz/__init__.py

```diff
@@ -12,3 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """libClusterfuzz package."""
+
+import os
+
+if not os.getenv('ROOT_DIR'):
+  # libClusterFuzz
+  os.environ['CONFIG_DIR_OVERRIDE'] = os.path.join(
+      os.path.dirname(os.path.abspath(__file__)), 'lib-config')
```

Comment:
nit: move os.path.dirname(os.path.abspath(__file__) to local and use in both calls.

Reply:
done!